test/examples/planar.js

```diff
@@ -14,4 +14,36 @@ describe('planar', () => {
         assert.ok(result);
         await page.close();
     });
+    it('should get picking position from depth', async function _() {
+        const page = await browser.newPage();
+
+        await page.setViewport({ width: 400, height: 300 });
+        await page.goto(`http://localhost:${itownsPort}/examples/planar_vector.html`);
+        await page.waitFor('#viewerDiv > canvas');
+        await exampleCanRenderTest(page, this.test.fullTitle());
```

Comment:
You could speed up the test by removing this. No need to wait for all textures to be loaded since you're going to move the camera anyway.